[2025-10-13T08:54:26.530+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T08:54:26.554+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_transactions manual__2025-10-13T08:53:56.784841+00:00 [queued]>
[2025-10-13T08:54:26.565+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_transactions manual__2025-10-13T08:53:56.784841+00:00 [queued]>
[2025-10-13T08:54:26.566+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T08:54:26.583+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_transactions> on 2025-10-13 08:53:56.784841+00:00
[2025-10-13T08:54:26.594+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=730) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T08:54:26.596+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_transactions', 'manual__2025-10-13T08:53:56.784841+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpxoqfdax5']
[2025-10-13T08:54:26.598+0000] {standard_task_runner.py:72} INFO - Started process 735 to run task
[2025-10-13T08:54:26.598+0000] {standard_task_runner.py:105} INFO - Job 111: Subtask extract_load_group.load_transactions
[2025-10-13T08:54:26.675+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_transactions manual__2025-10-13T08:53:56.784841+00:00 [running]> on host 84d5a327cf64
[2025-10-13T08:54:26.805+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_transactions' AIRFLOW_CTX_EXECUTION_DATE='2025-10-13T08:53:56.784841+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-13T08:53:56.784841+00:00'
[2025-10-13T08:54:26.806+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T08:54:36.212+0000] {load.py:30} INFO - Loaded 0 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_transactions
[2025-10-13T08:54:36.213+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T08:54:36.228+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T08:54:36.228+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_transactions, run_id=manual__2025-10-13T08:53:56.784841+00:00, execution_date=20251013T085356, start_date=20251013T085426, end_date=20251013T085436
[2025-10-13T08:54:36.301+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T08:54:36.325+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T08:54:36.328+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
